cypress/integration/CreateModels/AllEntityTypes.spec.js

```diff
@@ -0,0 +1,20 @@
+/**
+* Copyright (c) Microsoft Corporation. All rights reserved.  
+ * Licensed under the MIT License.
+*/
+
+const models = require('../../support/Models')
+const entities = require('../../support/Entities')
+
+describe('CreateModels', () => {
+  it('All Entity Types', () => {
```

Comment:
Can save for different review, but eventually want the `it` should say what the test is asserting.  Think keeping it as simple port is ok for now.

As some background on the naming. I can't remember the exact origins of the syntax but I think it is call Gherkin or "GivenWhenThen" meaning "Given state X when condition Y then expect Z" or something like that.  
Combining this with Mocha tooling and BDD style it's meant to read well to humans. `describe` is literally describing what the children tests are related to or declares the subject of the tests assume.  Then `it` s read into their phrase like "it should do x"

```
describe('myClass', () => {
  describe('doubleThings', () => {
    it('given a string it should return the string concatenated with itself', () => { ... })
    it('given a number should return double the number', () => { ... })
  })
})
```

Reply:
I understand that. Was intending to go that way moving forward, but not in this change set.